contracts/swappa/PairBPool.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.8;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "../interfaces/balancer/IBPool.sol";
+import "./ISwappaPairV1.sol";
+
+contract PairBPool is ISwappaPairV1 {
```

Comment:
for some reason, on this contract, I cannot declare the payable override function. when I declare that payable function, the build process fails...

maybe you can let me know what's wrong, contract works fine using default fallback though